src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Ball ball = new Ball();
+        int number = new Random().nextInt(100);
```

Comment:
100 should better be a constant field

https://www.javatpoint.com/java-constant